lib/models/manifest.rb

```diff
@@ -1,3 +1,56 @@
+require_relative '../query_dsl'
+
 class Manifest < ActiveRecord::Base
+  include Elasticsearch::Model
+
+  ActiveRecord::Base.raise_in_transactional_callbacks = true
+
+  after_commit on: [:create] do
```

Comment:
ok - then let's just make sure the methods using for this callback are private since we aren't using them outside of this class.